remotes/docker/config/hosts.go

```diff
@@ -22,6 +22,7 @@ import (
 	"crypto/tls"
 	"errors"
 	"fmt"
+	"io/ioutil"
```

Comment:
ioutil is deprecated

Reply:
ok, use `os.ReadDir` instead of `ioutil.ReadDir`